src/lib/datepicker/datepicker.spec.ts

```diff
@@ -486,6 +487,81 @@ describe('MdDatepicker', () => {
         expect(cells[1].classList).not.toContain('mat-calendar-body-disabled');
       });
     });
+
+    describe('datepicker with change and input events', () => {
+      let fixture: ComponentFixture<DatepickerWithChangeAndInputEvents>;
+      let testComponent: DatepickerWithChangeAndInputEvents;
+      let inputEl: HTMLInputElement;;
+
+      beforeEach(async(() => {
+        fixture = TestBed.createComponent(DatepickerWithChangeAndInputEvents);
+        fixture.detectChanges();
+
+        testComponent = fixture.componentInstance;
+        inputEl = fixture.debugElement.query(By.css('input')).nativeElement;
+
+        spyOn(testComponent, 'onChange');
```

Comment:
Oh, I guess you'd need to add them on the testComponent too anyway. So I guess it doesn't really help.